src/vsr/replica.zig

```diff
@@ -4943,11 +5104,9 @@ pub fn ReplicaType(
             switch (self.status) {
                 .normal => unreachable,
                 .view_change => {},
-                .recovering => {
-                    // The replica's view hasn't been set yet.
-                    // It will be set shortly, when we transition to normal status.
-                    assert(self.view == 0);
-                },
+                // The replica's view hasn't been set yet.
+                // It will be set shortly, when we transition to normal status.
+                .recovering => assert(self.view == 0),
```

Comment:
Random though: many of these "replica_count=1 never changes views" assertions won't be valid once we have reconfiguration, since you could reconfigure down to replicas=1 then back up. (Not that someone should ever do that. :fearful:)